website/content/docs/job-specification/proxy.mdx

```diff
@@ -79,10 +84,28 @@ sidecar_service {
 }
 ```
 
+The following example is a proxy specification that includes transparent proxy
+configuration. Note that with transparent proxy, you will not need to configure
+an `upstreams` block.
+
+```hcl
+sidecar_service {
+  proxy {
+    transparent_proxy {
+    }
+  }
+}
+```
+
+[Consul Connect]: /nomad/docs/integrations/consul-connect
 [job]: /nomad/docs/job-specification/job 'Nomad job Job Specification'
 [group]: /nomad/docs/job-specification/group 'Nomad group Job Specification'
 [task]: /nomad/docs/job-specification/task 'Nomad task Job Specification'
-[interpolation]: /nomad/docs/runtime/interpolation 'Nomad interpolation'
+[runtime variable interpolation]: /nomad/docs/runtime/interpolation 'Nomad interpolation'
 [sidecar_service]: /nomad/docs/job-specification/sidecar_service 'Nomad sidecar service Specification'
 [upstreams]: /nomad/docs/job-specification/upstreams 'Nomad upstream config Specification'
 [expose]: /nomad/docs/job-specification/expose 'Nomad proxy expose configuration'
+[envoy_dynamic_config]: /consul/docs/connect/proxies/envoy#dynamic-configuration
+[expose_path_ref]: /consul/docs/connect/registration/service-registration#expose-paths-configuration-reference
```

Comment:
this link is broken in the preview... I found [this one](https://nomad-git-f-tproxy-docs-hashicorp.vercel.app/consul/docs/connect/proxies/proxy-config-reference#expose-paths-configuration-reference) which seems like it might be right for this?


I also just happened to notice over on [the expose page](https://nomad-git-f-tproxy-docs-hashicorp.vercel.app/nomad/docs/job-specification/expose#path) there's a broken link for this same ref.  mind fixing that while you're in here?  this one: https://github.com/hashicorp/nomad/blob/v1.7.6/website/content/docs/job-specification/expose.mdx?plain=1#L136-L137

oh, and also in `upstreams.mdx` here: https://github.com/hashicorp/nomad/blob/v1.7.6/website/content/docs/job-specification/upstreams.mdx?plain=1#L85

I thiiink that's all of these?

```
$ rg 'connect/registration/service-registration' website/
website/content/docs/job-specification/expose.mdx
232:[consul-expose-path-config]: /consul/docs/connect/registration/service-registration#expose-paths-configuration-reference

website/content/docs/job-specification/proxy.mdx
109:[expose_path_ref]: /consul/docs/connect/registration/service-registration#expose-paths-configuration-reference

website/content/docs/job-specification/upstreams.mdx
85:  directly to Consul. See [Consul Connect documentation](/consul/docs/connect/registration/service-registration#upstream-configuration-reference)
```

Reply:
Will do!